teachertool/src/components/CatalogOverlay.tsx

```diff
@@ -80,7 +80,7 @@ const CatalogItem: React.FC<CatalogItemProps> = ({ catalogCriteria, recentlyAdde
     const { state: teacherTool } = useContext(AppStateContext);
 
     const existingInstanceCount = teacherTool.checklist.criteria.filter(
-        i => i.catalogCriteriaId === catalogCriteria.id
+        i => i.catalogCriteriaId === catalogCriteria.id && !i.deleted
```

Comment:
For long-term code maintainability, I recommend adding a helper (in state/helpers.ts) for getting active criteria.